xml/admin_saltcluster.xml

```diff
@@ -1141,67 +1142,87 @@ OSD_ID  HOST         STATE                    PG_COUNT  REPLACE  FORCE  STARTED_
    </step>
    <step>
     <para>
-     If the &adm; is a &smaster; at the same time, stop
-     <systemitem class="daemon">salt-master</systemitem>
-     <systemitem class="daemon">salt-minion</systemitem> services on the old
-     &adm;:
+     If the old &smaster; is also the only administration node in the cluster,
+     then manually move <filename>/etc/ceph/ceph.client.admin.keyring</filename>
+     and <filename>/etc/ceph/ceph.conf</filename> to the new &smaster;.
     </para>
+    </step>
+    <step>
+      <para>
+        Stop and disable the &smaster; &systemd; service on the old &smaster; node:
+      </para>
 <screen>
 &prompt.smaster;systemctl stop salt-master.service
+&prompt.smaster;systemctl disable salt-master.service
+</screen>
+   </step>
+   <step>
+     <para>
+       If the old &smaster; node is no longer in the cluster, also stop and
+       disable the &sminion; &systemd; service:
+     </para>
+<screen>
 &prompt.smaster;systemctl stop salt-minion.service
+&prompt.smaster;systemctl disable salt-minion.service
 </screen>
+    <warning>
+      <para>
+        Do not stop or disable the <literal>salt-minion.service</literal> if
+        the old &smaster; node has any &ceph; daemons (MON, MGR, OSD, MDS,
+        gateway, monitoring) running on it.
+      </para>
+    </warning>
    </step>
    <step>
     <para>
-     Install &cephos; on the new &adm; following the procedure described in
+     Install &cephos; on the new &smaster; following the procedure described in
      <xref linkend="deploy-os"/>.
     </para>
    </step>
    <step>
-    <para>
-     If the &adm; is a &smaster; at the same time, install the
-     <package>salt-master</package> (and possibly
-     <package>salt-minion</package>) package. If the &adm; is
-     <emphasis>not</emphasis> a &smaster;, install the
-     <package>salt-minion</package> package only. Configure &salt; so that the
-     &smaster; and &sminion;s communicate. Follow the procedure described in
-     <xref linkend="deploy-salt"/>.
-    </para>
-    <tip>
-     <title>Transition of Salt Minions</title>
      <para>
-      To ease the transition of &sminion;s to the new &adm;, remove the
-      original &smaster;'s public key from each of them:
+       Install the <package>salt-master</package> and <package>salt-minion</package>
```

Comment:
Installing `salt-minion` is not mandatory, can we make that explicit here? e.g.:
- `and <package>salt-minion</package> (if applicable)`
or
- `(and possibly <package>salt-minion</package>) package`

Reply:
How do you determine if it is applicable to install the package? This leaves it fairly vague.

Reply:
It's up to the administrator to decide if the salt-master is also a salt-minon or not.

In other words, it's up to the administrator do decide if the salt-master node will also run &ceph; daemons or not.

(we support both configurations)

Reply:
Makes sense. Will include your suggestion.